build.gradle

```diff
@@ -15,12 +15,14 @@ buildscript {
 allprojects {
 
     group 'com.github.jasync-sql'
-    //version '0.8.39-netty4.0'
-    version '0.8.39'
+    version '0.0.0'
```

Comment:
why reset the version number

Reply:
As you can see version number was not latest anyway. Wanted to prevent accidently overriding it.